pkg/console/subresource/configmap/configmap.go

```diff
@@ -142,6 +144,18 @@ func GetPluginsProxyServices(availablePlugins []*v1alpha1.ConsolePlugin) []conso
 	return proxyServices
 }
 
+func GetTelemetryConfiguration(consoleConfig *configv1.Console) map[string]string {
+	telemetry := make(map[string]string)
+	if len(consoleConfig.Annotations) > 0 {
+		for k, v := range consoleConfig.Annotations {
+			if strings.HasPrefix(k, "telemetry.console.openshift.io/") && len(k) > len("telemetry.console.openshift.io/") {
```

Comment:
lets have the `telemetry.console.openshift.io/` as a constant inside this package

Reply:
Updated: https://github.com/openshift/console-operator/compare/048ddfc2f096d78613716cb7203ec70e231aab4e..f8eb5fccdcf8e7bcfc594f6638807098c7a37ab5